model/project_ref.go

```diff
@@ -2303,6 +2303,19 @@ func (p *ProjectRef) UpdateAdminRoles(toAdd, toRemove []string) (bool, error) {
 		return false, nil
 	}
 	rm := evergreen.GetEnvironment().RoleManager()
+	projectCreateRole, err := rm.FindRoleWithPermissions(
```

Comment:
Can we add a test for this? We'd want to verify that we correctly find the role that's formatted like the one we have now, and that we correctly add it.